fix: add description for pages where the first line uses JS component (#327)
openfga/openfga.dev: docs/content/getting-started/perform-check.mdx

title: Perform a Check
sidebar_position: 4
slug: /getting-started/perform-check

> description: Perform a check request to determine whether a user has a certain relationship with an object.

Reviewer: ```suggestion
description: Check if a user is authorized to perform an action on a resource
```
